### ISSUE TITLE
Setting up autosplitter resulted in crash on computer startup

### ISSUE THREAD

**Issue by SushiInYourFace** [rhelmot/CelesteMagicTimer#15]
I was playing around with the autosplitter, trying to set it up for myself. In all honesty, I didn't know exactly what I was doing, but I'm pretty sure the result shouldn't have happened regardless. I changed the paths in celeste_tracer_loop_mine.c to point to where the autosplitter was stored in my drive, then ran the makefile. After some trial and error, I ran the executable file in the tracer folder (which said it connected to celeste.exe, but could not find class savedata). The next time I powered up the system, it would constantly crash on startup. The solution to this was booting to a USB, then deleting the CelesteMagicTimer folder on my drive.

TL;DR: I tried to set up the splitter, and it resulted in my system crashing on startup

**Comment by rhelmot** [comment]
Were you getting any error with a message originating from the tracer when you started up your computer? What mechanism were you using to add the tracer as a startup program?

**Comment by SushiInYourFace** [comment]
Sorry, I think I may have been a bit hasty in reporting this, as the system is now crashing again. I'm pretty sure now that removing the splitter didn't do anything, and it was instead something else about the live boot that helped me. Sorry for the false report!
